### ISSUE TITLE
Validation set generated but never used

### ISSUE THREAD

**Issue by saedr** [vuptran/graph-representation-learning#6]
Dear Phi, In the following piece of code you generate the validation set but never return it, instead you use the test set instead of validation set and there is no actual test set to be used for testing purposes.

https://github.com/vuptran/graph-representation-learning/blob/48d59781b1b66a3a639153a4eac5269cfd9b723c/longae/utils_gcn.py#L153-L172

Am I missing something?

**Comment by vuptran** [comment]
@saedr sorry for the really late response. You are correct, I don't use the val set, but subsume it into the training set. The test set (see below) is used for testing purposes. 

https://github.com/vuptran/graph-representation-learning/blob/master/longae/utils_gcn.py#L183
